core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitor.scala

```diff
@@ -278,6 +277,7 @@ class AvroUpsertMonitorActor(
     importFuture.map(_ => ImportComplete)  recover {
       case t =>
         logger.error(s"unexpected error in importFuture for ${attributes.importId}: ${t.getMessage}", t)
+        publishMessageToUpdateImportStatus(attributes.importId, None, ImportStatuses.Error, Option(t.getMessage))
```

Comment:
nit: since this error message is user-facing, we probably shouldn't rely just on the exception's message. How about:
```
        publishMessageToUpdateImportStatus(attributes.importId, None, ImportStatuses.Error,
          Option(s"Failed to import data. The underlying error message is: ${t.getMessage}"))
```